astroplan/core.py

```diff
@@ -2,6 +2,21 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord,
```

Comment:
I don't either, maybe @eteq can give us some hints on how to read these logs on Monday or in the Hangout on Tuesday.